.github/workflows/ci.yml

```diff
@@ -73,9 +73,9 @@ jobs:
           cache-dependency-path: ./packages/pangea-node-sdk/yarn.lock
 
       - name: Install dependencies
-        run: yarn install --frozen-lockfile --immutable
+        run: yarn install --frozen-lockfile
```

Comment:
Removed `--immutable`s since that's not a yarn v1 thing, someone I mixed stuff up.